membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -129,17 +128,34 @@ class AttributeController extends Controller with LoggingWithLogstashFields {
           subs
         }
       }
+
     }
 
-    val attributes: DisjunctionT[Future, String, Option[Attributes]] = for {
-      accounts <- EitherT(withTimer(s"ZuoraAccountIdsFromIdentityId", zuoraRestService.getAccounts(identityId)))
+    def zuoraAccountsQuery(identityId: String): Future[Disjunction[String, QueryResponse]] = zuoraRestService.getAccounts(identityId).map {
+      _.leftMap { error =>
+        log.error(s"Calling ZuoraAccountIdsFromIdentityId failed for identityId ${identityId}. Error: ${error}")
```

Comment:
error goes to sentry, so we just have to be careful in case some PII comes through in the error message (warn will only go to kibana and cloudwatch which is ok)
in this case I wouldn't log anything until it gets back to the Controller, just change to
```scala
_.leftMap { error =>
   s"Calling ZuoraAccountIdsFromIdentityId failed for identityId ${identityId}. Error: ${error}")
}
```